src/content/docs/codestream/observability/performance-monitoring.mdx

```diff
@@ -3,14 +3,14 @@ title: Observability in the IDE with CodeStream
 metaDescription: "Bring observability into the developer's IDE with New Relic CodeStream."
 redirects:
 - /docs/codestream/how-use-codestream/performance-monitoring
-freshnessValidatedDate: 2024-12-11
+freshnessValidatedDate: 2024-04-03
 ---
 
 import codestreamO11y from 'images/codestream_screenshot-crop_o11y.webp'
 
 New Relic CodeStream allows developers to see how the code they're responsible for is performing in production by bringing observability into the IDE and making it part of their daily routine. 
 
-Everything in CodeStream is contextual to the repository you have open in your IDE. If CodeStream doesn't know which services on New Relic are built from this repository, you may first be prompted to [associate the repository with a service](/docs/codestream/observability/repo-association). Otherwise, you'll get a comprehensive view of how the services built from your code are performing.
+Everything in CodeStream is contextual to the repositories you have open in your IDE. If CodeStream doesn't know which services on New Relic are built from any of the repositories, you may first be prompted to [associate the repository with a service](/docs/codestream/observability/repo-association). Otherwise, you'll get a comprehensive view of how the services built from your code are performing.
```

Comment:
```suggestion
In CodeStream, all aspects are dependent on the repositories you have open in your IDE. If CodeStream cannot identify the services on New Relic that are derived from any of the repositories, you will be prompted to [associate the repository with a service](/docs/codestream/observability/repo-association). Otherwise, you'll get a comprehensive view of how the services built from your code are performing.
```